lib/pliny/error_reporter.rb

```diff
@@ -14,29 +13,35 @@ def self.notify(exception, context: {}, rack_env: {})
     end
   end
 
-  class RollbarReporter
-    include ::Rollbar::ExceptionReporter
-    include ::Rollbar::RequestDataExtractor
+  begin
+    require 'rollbar/exception_reporter'
+    require 'rollbar/request_data_extractor'
 
-    def notify(exception, context:, rack_env:)
-      Rollbar.reset_notifier!
-      scope = fetch_scope(context: context, rack_env: rack_env)
-      Rollbar.scoped(scope) do
-        report_exception_to_rollbar(rack_env, exception)
+    class RollbarReporter
```

Comment:
okay, but what if not blowing up if rollbar is not installed? How does it get toggled off?

In the case of putting this code in a initializer put into the user's project, it's pretty simple: delete the initializer. 

The downside is that the more stuff in the template, the harder it is to track upstream additions to functionality.  I already have to do small backports to update Pliny versions.

 Perhaps it can live in a file that is "required" in the template explicitly to get the best of both worlds, so non-rollbar users can delete that "require" line?